### ISSUE TITLE
results different from Caffe SSD

### ISSUE THREAD

**Issue by miladf2** [seann999/ssd_tensorflow#10]
I compared the results of caffe detection code with the detection code implemented in this repo with the same caffe model (converted to tf model). The results are visually different. Boxes are smaller around objects in this tf implementation. There are some additional miss detection in TF. I think the pre processing code for images need to be refined. The images and annotations are probably not mapped accurately to 512*512 and back to original size. Just a guess!

**Comment by wcy940418** [comment]
Hello, @miladf2, it looks like you are succeeded in running this code. I have a problem when running function default_boxes, I always encountered segment fault. Did you see this problem? Thanks.

**Comment by miladf2** [comment]
@wcy940418 segmentation fault is most likely because u ran out of memory. try decreasing the batch_number and memory fraction.

**Comment by wcy940418** [comment]
@miladf2 Yes, you are right. Since I modified this SSD model to do something else, it seems like python list cannot handle such many defaults box data in my model. I changed python list data structure to numpy ndarray in defaults box, and it works well. But I encountered segmentation fault again when I invoke minimize loss function. I would try to reduce batch size, but do you think it is also related to batch size?
